apps/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -96,6 +102,10 @@ export abstract class BaseInstallAction extends BaseRushAction {
       }
     }
 
+    if (this._maxInstallAttempts.value && this._maxInstallAttempts.value < 1) {
```

Comment:
We probably should also throw if they provide `0`

Reply:
We throw if the value is less than `1`, that would cover `0`